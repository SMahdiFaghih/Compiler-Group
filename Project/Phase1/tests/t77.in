0x123 123 61 0xab 0x123Q32 0x123 45