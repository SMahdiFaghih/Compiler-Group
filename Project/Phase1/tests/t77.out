T_INTLITERAL 0x123
T_INTLITERAL 123
T_INTLITERAL 61
T_INTLITERAL 0xab
T_INTLITERAL 0x123
T_ID Q32
T_INTLITERAL 0x123
T_INTLITERAL 45